tests/accelerators/test_dp.py

```diff
@@ -127,17 +128,56 @@ def test_dp_test(tmpdir):
 
 @RunIf(min_gpus=2)
 def test_dp_training_step_dict(tmpdir):
-    """
-    This test verify dp properly reduce dictionaries
-    """
+    """ This test verifies that dp properly reduces dictionaries """
 
-    model = BoringModel()
+    class TestModel(BoringModel):
+
+        def train_dataloader(self):
+            return DataLoader(RandomDataset(32, 64), batch_size=2)
+
+        def val_dataloader(self):
+            return DataLoader(RandomDataset(32, 64), batch_size=2)
+
+        def test_dataloader(self):
+            return DataLoader(RandomDataset(32, 64), batch_size=2)
```

Comment:
isn't this already part of the model?

Reply:
I believe the difference is the batch size

Reply:
Yes exactly and because it's a DP test with 2 gpus we need at least batch size 2 for this test to be meaningful.